aspnet/fundamentals/hosting.rst

```diff
@@ -0,0 +1,297 @@
+:version: 1.0.0
+
+Hosting
+=======
+
+By `Steve Smith`_
+
+To run an ASP.NET Core app, you need to configure and launch a host using `WebHostBuilder <https://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNetCore/Hosting/WebHostBuilder/index.html?highlight=webhostbuilder>`_.
+
+.. contents:: Sections:
+  :local:
+  :depth: 1
+
+What is a Host?
+---------------
+
```

Comment:
The topic is 'Hosting' so it seemed appropriate to describe what that is, exactly. We don't spend much time on IWebHost, obviously, but it seemed like the logical place to begin (that, and disambiguating between Hosts and Servers, which is a common source of confusion among developers, IME).